### DISCUSSION TITLE
是否还招募维护人员

### DISCUSSION THREAD

**Discussion by Her-soulmate** [travellings-link/travellings#1552] [寻求帮助] [answered]
今天没事点了“开往”发现有很多网站无法访问，是人员不足、维护时间不够吗？
发现的几个无法访问的网站，已经通过docs提交；
如果还招募维护人员，我可以负责来排查网站的状态。

**Comment by linlinzzo** [comment, marked as answer]
还招。人员和维护时间的确还有不足的情况（主要是我们没有太多的精力频繁进行巡查）。

**Comment by linlinzzo** [comment]
如果你感兴趣，请通过即时通讯工具的方式联系我。
